pkg/controller/remoteingress/remoteingress_controller.go

```diff
@@ -58,12 +81,28 @@ func Add(mgr manager.Manager) error {
 // NewReconciler returns a new reconcile.Reconciler
 func NewReconciler(mgr manager.Manager) reconcile.Reconciler {
 	return &ReconcileRemoteClusterIngress{
-		Client: mgr.GetClient(),
-		scheme: mgr.GetScheme(),
-		logger: log.WithField("controller", controllerName),
+		Client:         mgr.GetClient(),
+		scheme:         mgr.GetScheme(),
+		logger:         log.WithField("controller", controllerName),
+		restConfig:     mgr.GetConfig(),
+		kubeCLIBuilder: kubeCLIBuilderFunc,
 	}
 }
 
+func kubeCLIBuilderFunc(config *rest.Config, logger log.FieldLogger) (kubeCLIApplier, error) {
+	var cliClient kubeCLIApplier
+	clientConfig := operatorutil.GenerateClientConfigFromRESTConfig("anything", config)
```

Comment:
Just fyi ... I'm adding another constructor for resource.Helper that takes a rest.Config in my PR. We should eventually switch to just one.